main.go

```diff
@@ -24,6 +26,13 @@ func main() {
 		Debug:   debug,
 	}
 
+	// Wire up ggcr logs.
+	logs.Warn.SetOutput(os.Stderr)
+	if debug {
+		logs.Progress.SetOutput(os.Stderr)
+		logs.Debug.SetOutput(os.Stderr)
+	}
```

Comment:
I wonder if there's something we should do to integrate with TF's tracing level stuff for this 🤔